ci/pipelines/prometheus-pushgateway.yml

```diff
@@ -37,6 +72,33 @@ resource_types:
       tag: latest
 
 jobs:
+  - name: push-adot-to-deploy-ecr
+    plan:
+      - get: adot-ecr-registry-staging
+        params:
+          format: oci
+        trigger: true
+      - load_var: application_image_tag
+        file: adot-ecr-registry-staging/tag
+      - get: pay-ci
+      - task: parse-staging-release-tag
+        file: pay-ci/ci/tasks/parse-staging-release-tag.yml
+        input_mapping:
+          ecr-repo: adot-ecr-registry-staging
+      - load_var: parse_staging_release_tag
+        file: parse-staging-release-tag/tag
+      - task: send-job-release-info-to-hosted-graphite
```

Comment:
Can you remove this task, it's for finding problems in the main deploy-to pipelines. You can also remove the get from pay-ci, the parse release tags jobs too by removing this task